hooks/command

```diff
@@ -230,7 +230,7 @@ config_path="config.json"
 prerequisites="$(plugin_read_list SIGN_PREREQUISITES)"
 if [[ -z "${prerequisites}" ]]; then
   # If no prerequisites are specified, point to the artifact
-  prerequisites="${unsigned_artifact}"
+  prerequisites="${base_artifact}"
```

Comment:
This was the primary issue; everything else is cleanup.

Tested on our signing agent, works fine.